README.md

```diff
@@ -111,6 +111,24 @@ $ kubectl get pod -l app=example-appservice
 NAME                     READY     STATUS    RESTARTS   AGE
 example-appservice-pod   1/1       Running   0          1m
 
+# Test the new Resource Type
+$ kubectl describe appservice
```

Comment:
Nit: just have one minor suggestion
```suggestion
$ kubectl describe appservice example-appservice
```

Reply:
Just a friendly ping @jonnyfiveiq in case you missed the above suggestion, thanks for the PR!

Reply:
@LiliC thanks, all good with me.

Reply:
@jonnyfiveiq Sorry, didn't make myself clear enough, do you mind accepting the suggestions :) Thanks!